install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-trusty

```diff
@@ -12,13 +12,13 @@ securedrop-app-code (0.12.2+trusty) trusty; urgency=medium
 
 securedrop-app-code (0.12.1+trusty) trusty; urgency=medium
 
-  * See changelog.md 
+  * See changelog.md
```

Comment:
these changelog files I think are accidentally included here

Reply:
[I've removed these whitespace changes in the changelog and rebased and squashed my commit]